 Common/sources/license.js | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/Common/sources/license.js b/Common/sources/license.js
index 8813cbac..2db51eff 100644
--- a/Common/sources/license.js
+++ b/Common/sources/license.js
@@ -44,23 +44,23 @@ exports.readLicense = async function () {
 	return [{
 		count: 1,
 		type: c_LR.Success,
-		packageType: constants.PACKAGE_TYPE_OS,
+		packageType: constants.PACKAGE_TYPE_I,
 		mode: constants.LICENSE_MODE.None,
-		branding: false,
+		branding: true,
 		connections: constants.LICENSE_CONNECTIONS,
 		connectionsView: constants.LICENSE_CONNECTIONS,
-		customization: false,
-		advancedApi: false,
+		customization: true,
+		advancedApi: true,
 		usersCount: 0,
 		usersViewCount: 0,
 		usersExpire: constants.LICENSE_EXPIRE_USERS_ONE_DAY,
-		hasLicense: false,
+		hasLicense: true,
 		buildDate: oBuildDate,
 		startDate: startDate,
-		endDate: null,
+		endDate: new Date("2099-01-01T23:59:59.000Z"),
 		customerId: "",
-		alias: ""
+		alias: "community"
 	}, null];
 };
 
-exports.packageType = constants.PACKAGE_TYPE_OS;
+exports.packageType = constants.PACKAGE_TYPE_I;
diff --git a/DocService/sources/server.js b/DocService/sources/server.js
index 09c9e3bf..8b5f4948 100644
--- a/DocService/sources/server.js
+++ b/DocService/sources/server.js
@@ -129,7 +129,7 @@ if (!(cfgTokenEnableBrowser && cfgTokenEnableRequestInbox && cfgTokenEnableReque
 }
 
 updateLicense();
-fs.watchFile(cfgLicenseFile, updateLicense);
+
 setInterval(updateLicense, 86400000);
 
 try {
diff --git a/FileConverter/sources/convertermaster.js b/FileConverter/sources/convertermaster.js
index 46615032..ef71ae9e 100644
--- a/FileConverter/sources/convertermaster.js
+++ b/FileConverter/sources/convertermaster.js
@@ -92,7 +92,7 @@ if (cluster.isMaster) {
 
   updateLicense();
 
-  fs.watchFile(cfgLicenseFile, updateLicense);
+
   setInterval(updateLicense, 86400000);
 } else {
   const converter = require('./converter');
diff --git a/Makefile b/Makefile
index e8e1308f..23f7e2ef 100644
--- a/Makefile
+++ b/Makefile
@@ -87,7 +87,7 @@ DEBUG = $(BRANDING_DIR)/debug.js
 .PHONY: all clean install uninstall build-date
 
 .NOTPARALLEL:
-all: $(SPELLCHECKER_DICTIONARIES) $(TOOLS) $(SCHEMA) $(CORE_FONTS) $(DOCUMENT_TEMPLATES) $(LICENSE) $(WELCOME) $(INFO) build-date
+all: $(SCHEMA) $(LICENSE) $(WELCOME) $(INFO) build-date
 
 build-date: $(GRUNT_FILES)
 	sed "s|\(const buildVersion = \).*|\1'${PRODUCT_VERSION}';|" -i $(COMMON_DEFINES_JS)
